math/vc_soa/include/algebra/math/vc_soa.hpp

```diff
@@ -0,0 +1,14 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2020-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+// clang-format off
+#include "algebra/math/impl/vc_soa_vector.hpp"
+#include "algebra/math/impl/vc_soa_getter.hpp"
```

Comment:
It's not great that the two would have a fixed order for being included. :frowning: Why is that?

Reply:
I need the dot product in the getter functionality, I just included the header now